### PR TITLE
travis: PHP 7.0 nightly added, consistent indenting

### PR DESCRIPTION
**Pull request by TomasVotruba**
Ref #199

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,15 +11,17 @@ php:
...
 
 install: 
-    - composer install --dev
-    - "[ $TRAVIS_PHP_VERSION = 'hhvm' ] || composer require 'doctrine/mongodb-odm' '*@beta'"
+  - composer install --dev
+  - "if [[ $TRAVIS_PHP_VERSION != 'hhvm' && $TRAVIS_PHP_VERSION != '7.0' ]]; then composer require 'doctrine/mongodb-odm' '*@beta'; fi"
 
-script: phpunit -v
+script:
+  - phpunit
```
Comment (stof) [inline .travis.yml]: Please keep PHPUnit in verbose mode, which explains why things get skipped

Comment (TomasVotruba) [inline .travis.yml]: I see.

Comment (TomasVotruba) [inline .travis.yml]: Consistency over Doctrine repositories would be great!

Comment (Ocramius) [inline .travis.yml]: @TomasVotruba you can't ask for that if separate maintainers work on separate repos.

Comment (TomasVotruba) [inline .travis.yml]: @Ocramius I see.

**Comment by Ocramius** [comment]
:+1:

**Comment by TomasVotruba** [comment]
Thanks

**Pull request merged**